include/boost/geometry/algorithms/append.hpp

```diff
@@ -60,7 +61,7 @@ struct to_range_point
     static inline void apply(Geometry& geometry, Point const& point,
                              signed_size_type = -1, signed_size_type = 0)
     {
-        typename geometry::point_type<Geometry>::type copy;
+        geometry::point_type_t<Geometry> copy;
```

Comment:
This first confused me.
Because, what about `point_t`  ...

But it is correct.

From the standard: 
`using add_pointer_t = typename add_pointer<T>::type`
`using common_type_t = typename common_type<T...>::type`

We have to confirm to the second form.

Reply:
Yes, the standard pattern unfortunately is different and `point_t` would probably be confusing.

On similar note. I was thinking about adding aliases for Boost.Range traits like `typename boost::range_value<Range>::type` in `util/range.hpp`. C++20 adds this as `std::ranges::range_value_t<>`. In Boost.Geometry there is `namespace bg::range` already. This seems to be the best place for these aliases even though the word `range` is repeated. However I don't have a strong opinion about it, I see several possibilities:
- `boost::geometry::range::range_value_t<>`
- `boost::geometry::range_value_t<>`
- `boost::geometry::range::value_t<>`
- `boost::geometry::ranges::range_value_t<>` (probably the closest to the standard)

Do you have any opinions?

Reply:
I would adhere to the standard. So that's the last one (incl. namespace) or the first one (with our namespace).

Reply:
I also agree with the 1st or last.